PavlenkoHW/src/HW01052020/Revers.java

```diff
@@ -0,0 +1,19 @@
+package HW01052020;
+
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * 5. Написать функцию, которая реверсирует список, возвращает список элементов
+ * в обратном порядке. {1, 2, 3, 4, 5} -> {5, 4, 3, 2, 1}
+ */
+public class Revers {
+
+    public static List reversList(List list) {
+        LinkedList retList = new LinkedList();
+        for (int i = 0; i < list.size(); i++) {
+            retList.addFirst(list.get(i));
+        }
+        return retList;
```

Comment:
I think there is a reverse() function somewhere in the collection api.

Reply:
in this case, man can write a function Collections.reverce(list) directly in the test)